### DISCUSSION TITLE
Live View Netz aktualisiert nicht (Hichi über HTTP + JSON)

### DISCUSSION THREAD

**Discussion by pelletq** [hoylabs/OpenDTU-OnBattery#722] [General]
Hi,

//war ein bekanntes Problem der aktuellen Firmware und ist bereits in Bearbeitung/behoben für den nächsten Release. Thema kann somit ignoriert werden.

ich habe heute mal auf die aktuelle Firmware geupdatet via OTA. Vorhin ist mir dann aufgefallen, dass sich im Live View die Anzeige für den Netzbezug nicht mehr aktualisiert. Der Inverter wird aber gleichzeitig weiterhin an den aktuellen Verbrauch angepasst - die Werte scheinen also anzukommen. 

Dafür spricht auch, dass bei Aktualisierung der Seite dann auch der aktualisierte Wert angezeigt wird - und dann wieder stehen bleibt.
Unter Power Meter habe ich keine Änderungen vorgenommen und der Test gibt erfolgreich den aktuellen Verbrauch wieder 
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/152500090/97c8222c-ccb8-45fa-8c0c-7f589cc6a4a4)

In der Konsole fällt mir sowas hier auf, das ist mir vorher nicht wissentlich untergekommen: 

22:02:11.344 > Websocket: [/livedata][23] connect
22:02:11.449 > Websocket: [/vedirectlivedata][24] connect
22:02:11.591 > Websocket: [/batterylivedata][23] connect
22:02:11.675 > Websocket: [/livedata][22] disconnect
22:02:11.742 > Websocket: [/vedirectlivedata][23] disconnect
22:02:11.831 > Websocket: [/batterylivedata][22] disconnect

Dabei sind die anderen Live-Daten sichtbar und ändern sich auch alle ~5 Sekunden (WR, MPPT, Batterie).
Sowohl über 2 Browser am PC als auch am Handy kommt das so vor, sollte also nicht am Endgerät liegen.

Beispiel:
22: 02 Ich öffne den LiveView. Netzbetzug gibt 89 Watt und bleibt darauf stehen (die direkt darauf folgende Änderung auf 126 Watt sehe ich nicht)
In der Konsole läuft unten stehendes (stark gekürzt, da ich 2 Minuten gewartet habe)
22:03:30 Ich drücke F5, der Live View zeigt 72 Watt bei Netzbezug an (die direkt darauf folgende Änderung auf 67 Watt sehe ich nicht)

`PowerMeterClass: TotalPower: 89.00
22:02:11.344 > Websocket: [/livedata][23] connect
22:02:11.449 > Websocket: [/vedirectlivedata][24] connect
22:02:11.591 > Websocket: [/batterylivedata][23] connect
22:02:11.675 > Websocket: [/livedata][22] disconnect
22:02:11.742 > Websocket: [/vedirectlivedata][23] disconnect
22:02:11.831 > Websocket: [/batterylivedata][22] disconnect
22:02:11.901 > Fetch inverter: 116491627608
22:02:11.991 > TX RealTimeRunData 865.00 MHz --> 15 91 62 76 08 80 19 96 80 80 0B 00 65 E8 D9 D3 00 00 00 00 00 00 00 00 56 0D 40 
22:02:12.114 > Interrupt received
22:02:12.169 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 01 00 01 01 E5 01 E5 00 91 00 92 02 BF 02 C7 00 00 EC | -14 dBm
22:02:12.300 > Interrupt received
22:02:12.390 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 02 32 1B 00 00 32 98 03 94 03 9D 01 E6 01 E6 00 8C 93 | -15 dBm
22:02:12.500 > Interrupt received
22:02:12.571 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 03 00 94 02 AA 02 D3 00 00 30 41 00 00 33 55 03 7A 17 | -14 dBm
22:02:12.635 > Interrupt received
22:02:12.701 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 04 03 B0 09 25 13 83 0A 77 00 02 00 72 03 E7 00 CB BE | -15 dBm
22:02:12.763 > Interrupt received
22:02:12.836 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 85 00 BB 2B 3C BE | -15 dBm
22:02:12.894 > RX Period End
22:02:13.028 > Success
22:02:13.124 > [DPL::loop] ******************* ENTER **********************
22:02:13.195 > [DPL::loop] battery interface enabled, SoC: 46 %, StartTH: 50 %, StopTH: 15 %, SoC age: 0 s, ignore: no
22:02:13.269 > [DPL::getBatteryVoltage] BMS: 48.42 V, MPPT: 48.45 V, inverter: 48.50 V, returning: 48.42V
22:02:13.327 > [DPL::loop] dcVoltage: 48.42 V, loadCorrectedVoltage: 48.77 V, StartTH: 50.00 V, StopTH: 49.00 V
22:02:13.535 > [DPL::loop] StartTH reached: no, StopTH reached: no, inverter is producing
22:02:13.642 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
22:02:13.744 > [DPL::loop] battery discharging allowed, PowerMeter: 126 W, target consumption: 30 W
22:02:13.833 > [DPL::setNewPowerLimit] requested: 363 W, (re-)sending limit: 363 W
22:02:13.899 > [DPL::loop] ******************* Leaving PL, calculated limit: 363 W, requested limit: 363 W (updated from calculated)
22:02:14.043 > TX ActivePowerControl 865.00 MHz --> 51 91 62 76 08 80 19 96 80 81 0B 00 0E 2E 00 00 81 63 1B 
22:02:14.360 > [   1781.513] DPL: waiting for a power limit command to complete
22:02:14.460 > Interrupt received
22:02:14.557 > RX 865.00 MHz --> D1 91 62 76 08 80 19 96 80 81 00 00 0B 00 14 07 4A | -14 dBm
22:02:14.783 > RX Period End
22:02:14.869 > Success
22:02:14.962 > [   1783.520] DPL: waiting for the system to settle
22:02:16.883 > Fetch inverter: 116491627608
22:02:16.952 > TX RealTimeRunData 865.00 MHz --> 15 91 62 76 08 80 19 96 80 80 0B 00 65 E8 D9 D8 00 00 00 00 00 00 00 00 66 7E 08 
22:02:17.109 > Interrupt received
22:02:17.215 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 01 00 01 01 E4 01 E5 00 B5 00 B7 03 6C 03 77 00 00 8F | -14 dBm
22:02:17.279 > Interrupt received
22:02:17.452 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 02 32 1B 00 00 32 98 03 94 03 9D 01 E5 01 E5 00 B0 AF | -14 dBm
22:02:17.513 > Interrupt received
22:02:17.581 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 03 00 BA 03 56 03 87 00 00 30 41 00 00 33 55 03 7A 91 | -14 dBm
22:02:17.666 > Interrupt received
22:02:17.726 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 04 03 B0 09 29 13 84 0D 12 00 02 00 8E 03 E7 00 CB 2B | -15 dBm
22:02:17.791 > Interrupt received
22:02:18.033 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 85 00 BB 76 65 BA | -14 dBm
22:02:18.085 > RX Period End
22:02:18.141 > Success
22:02:18.221 > [   1786.517] DPL: waiting for sufficiently recent inverter data
22:02:20.976 > PowerMeterClass: TotalPower: 126.00
22:02:21.875 > Fetch inverter: 116491627608
22:02:22.146 > TX RealTimeRunData 865.00 MHz --> 15 91 62 76 08 80 19 96 80 80 0B 00 65 E8 D9 DD 00 00 00 00 00 00 00 00 36 41 62 
22:02:22.228 > Interrupt received
22:02:22.373 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 01 00 01 01 E4 01 E4 00 B4 00 B6 03 6B 03 75 00 00 8B | -14 dBm
22:02:22.511 > Interrupt received
22:02:22.641 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 02 32 1B 00 00 32 98 03 94 03 9D 01 E5 01 E5 00 B0 AF | -14 dBm
22:02:22.692 > Interrupt received
22:02:22.849 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 03 00 BA 03 56 03 87 00 00 30 41 00 00 33 55 03 7A 91 | -14 dBm
22:02:22.922 > Interrupt received
22:02:23.002 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 04 03 B0 09 2A 13 84 0D 0E 00 00 00 8E 03 E8 00 CB 39 | -15 dBm
22:02:23.066 > Interrupt received
22:02:23.130 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 85 00 BB ED F5 B1 | -15 dBm
22:02:23.183 > RX Period End
22:02:23.321 > Success
22:02:23.393 > [DPL::loop] ******************* ENTER **********************
22:02:23.454 > [DPL::loop] battery interface enabled, SoC: 46 %, StartTH: 50 %, StopTH: 15 %, SoC age: 0 s, ignore: no
22:02:23.560 > [DPL::getBatteryVoltage] BMS: 48.37 V, MPPT: 48.37 V, inverter: 48.40 V, returning: 48.37V
22:02:23.612 > [DPL::loop] dcVoltage: 48.37 V, loadCorrectedVoltage: 48.80 V, StartTH: 50.00 V, StopTH: 49.00 V
22:02:23.682 > [DPL::loop] StartTH reached: no, StopTH reached: no, inverter is producing
22:02:23.747 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
22:02:24.073 > [DPL::loop] battery discharging allowed, PowerMeter: 126 W, target consumption: 30 W
22:02:24.127 > [DPL::setNewPowerLimit] requested: 430 W, (re-)sending limit: 430 W
22:02:24.189 > [DPL::loop] ******************* Leaving PL, calculated limit: 430 W, requested limit: 430 W (updated from calculated)
22:02:24.286 > TX ActivePowerControl 865.00 MHz --> 51 91 62 76 08 80 19 96 80 81 0B 00 10 CC 00 00 9F C5 5F 
22:02:24.356 > [   1791.845] DPL: waiting for a power limit command to complete
22:02:24.594 > Interrupt received
22:02:24.665 > RX 865.00 MHz --> D1 91 62 76 08 80 19 96 80 81 00 00 0B 00 14 07 4A | -14 dBm
22:02:25.116 > RX Period End
22:02:25.515 > Success
22:02:25.618 > [   1793.853] DPL: waiting for the system to settle
22:02:26.888 > Fetch inverter: 116491627608
22:02:26.960 > TX RealTimeRunData 865.00 MHz --> 15 91 62 76 08 80 19 96 80 80 0B 00 65 E8 D9 E2 00 00 00 00 00 00 00 00 C7 54 B9 
22:02:27.041 > Interrupt received
22:02:27.105 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 01 00 01 01 E3 01 E3 00 D9 00 DB 04 19 04 27 00 00 AB | -15 dBm
22:02:27.170 > Interrupt received
22:02:27.251 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 02 32 1B 00 00 32 98 03 94 03 9D 01 E4 01 E4 00 D4 CB | -15 dBm
22:02:27.356 > Interrupt received
22:02:27.429 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 03 00 E0 04 05 04 3E 00 00 30 41 00 00 33 56 03 7A 22 | -14 dBm
22:02:27.516 > Interrupt received
22:02:27.658 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 04 03 B1 09 2D 13 83 0F B0 00 03 00 AA 03 E7 00 CB AC | -15 dBm
22:02:27.714 > Interrupt received
22:02:27.798 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 85 00 BB AB E7 E5 | -14 dBm
22:02:27.968 > RX Period End
22:02:28.065 > Success
22:02:28.126 > [   1796.848] DPL: waiting for sufficiently recent inverter data
22:02:31.528 > PowerMeterClass: TotalPower: 34.00
22:02:31.889 > Fetch inverter: 116491627608
22:02:32.079 > TX RealTimeRunData 865.00 MHz --> 15 91 62 76 08 80 19 96 80 80 0B 00 65 E8 D9 E7 00 00 00 00 00 00 00 00 97 6B D3 
22:02:32.200 > Interrupt received
22:02:32.271 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 01 00 01 01 E3 01 E3 00 D5 00 D7 04 05 04 12 00 00 82 | -14 dBm
22:02:32.367 > Interrupt received
22:02:32.470 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 02 32 1B 00 00 32 98 03 94 03 9D 01 E4 01 E4 00 D0 CF | -14 dBm
22:02:32.531 > Interrupt received
22:02:32.690 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 03 00 DC 03 F1 04 29 00 00 30 41 00 00 33 56 03 7A FA | -14 dBm
22:02:32.775 > Interrupt received
22:02:32.876 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 04 03 B1 09 2D 13 84 0F 64 00 00 00 A7 03 E7 00 CB 71 | -15 dBm
22:02:32.939 > Interrupt received
22:02:33.003 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 85 00 BB 04 7D D0 | -15 dBm
22:02:33.066 > RX Period End
22:02:33.188 > Success
22:02:33.303 > [DPL::loop] ******************* ENTER **********************
22:02:33.382 > [DPL::loop] battery interface enabled, SoC: 46 %, StartTH: 50 %, StopTH: 15 %, SoC age: 0 s, ignore: no
22:02:33.437 > [DPL::getBatteryVoltage] BMS: 48.34 V, MPPT: 48.27 V, inverter: 48.30 V, returning: 48.34V
22:02:33.498 > [DPL::loop] dcVoltage: 48.34 V, loadCorrectedVoltage: 48.85 V, StartTH: 50.00 V, StopTH: 49.00 V
22:02:33.560 > [DPL::loop] StartTH reached: no, StopTH reached: no, inverter is producing
22:02:33.666 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
22:02:33.726 > [DPL::loop] battery discharging allowed, PowerMeter: 34 W, target consumption: 30 W
22:02:33.781 > [DPL::setNewPowerLimit] requested: 398 W, (re-)sending limit: 398 W
22:02:33.916 > [DPL::loop] ******************* Leaving PL, calculated limit: 398 W, requested limit: 398 W (updated from calculated)
22:02:33.978 > TX ActivePowerControl 865.00 MHz --> 51 91 62 76 08 80 19 96 80 81 0B 00 0F 8C 00 00 9F C3 06 
22:02:34.041 > [   1801.777] DPL: waiting for a power limit command to complete

...
...

22:03:29.556 > [DPL::loop] ******************* ENTER **********************
22:03:29.712 > [DPL::loop] battery interface enabled, SoC: 45 %, StartTH: 50 %, StopTH: 15 %, SoC age: 1 s, ignore: no
22:03:29.826 > [DPL::getBatteryVoltage] BMS: 48.44 V, MPPT: 48.38 V, inverter: 48.40 V, returning: 48.44V
22:03:29.940 > [DPL::loop] dcVoltage: 48.44 V, loadCorrectedVoltage: 48.85 V, StartTH: 50.00 V, StopTH: 49.00 V
22:03:30.080 > [DPL::loop] StartTH reached: no, StopTH reached: no, inverter is producing
22:03:30.140 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
22:03:30.208 > [DPL::loop] battery discharging allowed, PowerMeter: 72 W, target consumption: 30 W
22:03:30.427 > [DPL::setNewPowerLimit] requested: 360 W, last limit: 348 W, diff: 12 W, hysteresis: 30 W, age: 25553 ms
22:03:30.635 > [DPL::loop] ******************* Leaving PL, calculated limit: 360 W, requested limit: 348 W (kept last requested)
22:03:30.714 > Interrupt received
22:03:30.779 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 03 00 AC 03 16 03 44 00 00 30 43 00 00 33 57 03 7C 02 | -13 dBm
22:03:30.839 > RX Period End
22:03:30.938 > Last missing
22:03:31.048 > Request retransmit: 4
22:03:31.157 > TX RequestFrame 865.00 MHz --> 15 91 62 76 08 80 19 96 80 84 93 
22:03:31.291 > Interrupt received
22:03:31.381 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 04 03 B2 09 2C 13 83 0C 18 00 02 00 83 03 E7 00 CC 2A | -14 dBm
22:03:31.474 > RX Period End
22:03:31.914 > Last missing
22:03:31.985 > Request retransmit: 5
22:03:32.061 > TX RequestFrame 865.00 MHz --> 15 91 62 76 08 80 19 96 80 85 92 
22:03:32.122 > Interrupt received
22:03:32.182 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 85 00 BB D0 D4 AD | -14 dBm
22:03:32.245 > RX Period End
22:03:32.414 > Middle missing
22:03:32.493 > Request retransmit: 2
22:03:32.558 > TX RequestFrame 865.00 MHz --> 15 91 62 76 08 80 19 96 80 82 95 
22:03:32.656 > Interrupt received
22:03:32.784 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 02 32 1C 00 00 32 99 03 95 03 9E 01 E6 01 E5 00 A2 BA | -14 dBm
22:03:32.882 > RX Period End
22:03:33.092 > Success
22:03:33.168 > [DPL::loop] ******************* ENTER **********************
22:03:33.233 > [DPL::loop] battery interface enabled, SoC: 45 %, StartTH: 50 %, StopTH: 15 %, SoC age: 0 s, ignore: no
22:03:33.320 > [DPL::getBatteryVoltage] BMS: 48.41 V, MPPT: 48.40 V, inverter: 48.40 V, returning: 48.41V
22:03:33.385 > [DPL::loop] dcVoltage: 48.41 V, loadCorrectedVoltage: 48.81 V, StartTH: 50.00 V, StopTH: 49.00 V
22:03:33.449 > [DPL::loop] StartTH reached: no, StopTH reached: no, inverter is producing
22:03:33.524 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
22:03:33.592 > [DPL::loop] battery discharging allowed, PowerMeter: 72 W, target consumption: 30 W
22:03:33.716 > [DPL::setNewPowerLimit] requested: 351 W, last limit: 348 W, diff: 3 W, hysteresis: 30 W, age: 26809 ms
22:03:33.807 > [DPL::loop] ******************* Leaving PL, calculated limit: 351 W, requested limit: 348 W (kept last requested)
22:03:33.862 > Websocket: [/livedata][23] disconnect
22:03:33.916 > Websocket: [/vedirectlivedata][24] disconnect
22:03:34.024 > Websocket: [/batterylivedata][23] disconnect
22:03:34.106 > [DPL::loop] ******************* ENTER **********************
22:03:34.216 > [DPL::loop] battery interface enabled, SoC: 45 %, StartTH: 50 %, StopTH: 15 %, SoC age: 1 s, ignore: no
22:03:34.303 > [DPL::getBatteryVoltage] BMS: 48.41 V, MPPT: 48.41 V, inverter: 48.40 V, returning: 48.41V
22:03:34.396 > [DPL::loop] dcVoltage: 48.41 V, loadCorrectedVoltage: 48.81 V, StartTH: 50.00 V, StopTH: 49.00 V
22:03:34.478 > [DPL::loop] StartTH reached: no, StopTH reached: no, inverter is producing
22:03:34.558 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
22:03:34.637 > Websocket: [/livedata][24] connect
22:03:34.723 > [DPL::loop] battery discharging allowed, PowerMeter: 67 W, target consumption: 30 W
22:03:34.786 > [DPL::setNewPowerLimit] requested: 346 W, last limit: 348 W, diff: 2 W, hysteresis: 30 W, age: 28094 ms
22:03:35.034 > [DPL::loop] ******************* Leaving PL, calculated limit: 346 W, requested limit: 348 W (kept last requested)
22:03:35.114 > Websocket: [/vedirectlivedata][25] connect
22:03:35.191 > Websocket: [/batterylivedata][24] connect
22:03:35.255 > [DPL::loop] ******************* ENTER **********************
22:03:35.449 > [DPL::loop] battery interface enabled, SoC: 45 %, StartTH: 50 %, StopTH: 15 %, SoC age: 0 s, ignore: no
22:03:35.500 > [DPL::getBatteryVoltage] BMS: 48.39 V, MPPT: 48.35 V, inverter: 48.40 V, returning: 48.39V
22:03:35.569 > [DPL::loop] dcVoltage: 48.39 V, loadCorrectedVoltage: 48.79 V, StartTH: 50.00 V, StopTH: 49.00 V
22:03:35.639 > [DPL::loop] StartTH reached: no, StopTH reached: no, inverter is producing
22:03:35.698 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
22:03:35.762 > [DPL::loop] battery discharging allowed, PowerMeter: 67 W, target consumption: 30 W
22:03:35.833 > [DPL::setNewPowerLimit] requested: 346 W, last limit: 348 W, diff: 2 W, hysteresis: 30 W, age: 29328 ms
22:03:35.897 > [DPL::loop] ******************* Leaving PL, calculated limit: 346 W, requested limit: 348 W (kept last requested)
22:03:35.966 > PowerMeterClass: TotalPower: 67.00
22:03:36.061 > Fetch inverter: 116491627608
22:03:36.156 > [DPL::loop] ******************* ENTER **********************
22:03:36.215 > [DPL::loop] battery interface enabled, SoC: 45 %, StartTH: 50 %, StopTH: 15 %, SoC age: 2 s, ignore: no
22:03:36.364 > [DPL::getBatteryVoltage] BMS: 48.39 V, MPPT: 48.39 V, inverter: 48.40 V, returning: 48.39V
22:03:36.469 > [DPL::loop] dcVoltage: 48.39 V, loadCorrectedVoltage: 48.79 V, StartTH: 50.00 V, StopTH: 49.00 V
22:03:36.586 > [DPL::loop] StartTH reached: no, StopTH reached: no, inverter is producing
22:03:36.659 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
22:03:36.987 > [DPL::loop] battery discharging allowed, PowerMeter: 67 W, target consumption: 30 W
22:03:37.056 > [DPL::setNewPowerLimit] requested: 346 W, last limit: 348 W, diff: 2 W, hysteresis: 30 W, age: 30495 ms
22:03:37.159 > [DPL::loop] ******************* Leaving PL, calculated limit: 346 W, requested limit: 348 W (kept last requested)
22:03:37.219 > [   1863.248] DPL: the system is stable, the last power limit is still valid
22:03:37.292 > TX RealTimeRunData 865.00 MHz --> 15 91 62 76 08 80 19 96 80 80 0B 00 65 E8 DA 26 00 00 00 00 00 00 00 00 0D C6 26 
22:03:37.342 > Interrupt received
22:03:37.396 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 01 00 01 01 E4 01 E4 00 B8 00 BA 03 7D 03 87 00 00 6F | -14 dBm
22:03:37.505 > Interrupt received
22:03:37.695 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 02 32 1C 00 00 32 99 03 95 03 9E 01 E5 01 E5 00 B3 A8 | -14 dBm
22:03:37.799 > Interrupt received
22:03:37.907 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 03 00 BE 03 68 03 99 00 00 30 43 00 00 33 57 03 7C B3 | -15 dBm
22:03:37.966 > Interrupt received
22:03:38.108 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 04 03 B2 09 2C 13 84 0D 53 00 00 00 91 03 E8 00 CC 78 | -14 dBm
22:03:38.170 > Interrupt received
22:03:38.233 > RX 865.00 MHz --> 95 91 62 76 08 80 19 96 80 85 00 BB 96 20 1F | -14 dBm
22:03:38.314 > RX Period End
22:03:38.418 > Success

`

**Comment by spcqike** [comment]
#685 , #682 

Das Problem ist in der aktuellen develop Version bereits behoben, aber noch nicht im main branch.

**Reply by pelletq** [reply]
Danke und sorry für das offensichtlich doppelt- und dreifache Thema. Habe das ganze Forum durchsucht - allerdings nicht die Issues.

**Reply by hoschiking** [reply]
Na dann wird mal langsam Zeit für ein neues Release ;-)

**Reply by mastertape** [reply]
Ja das wäre sehr praktisch ....
